include/kremlin/internal/types.h

```diff
@@ -49,19 +49,21 @@ typedef const char *Prims_string;
 
 /* The great static header headache. */
 
+#define IS_MSVC64 (defined(_MSC_VER) && defined(_M_X64) && !defined(__clang__))
+#define HAS_INT128 ((defined(__x86_64__) || defined(__x86_64) || \
+                     defined(__aarch64__) || \
+                     (defined(__powerpc64__) && defined(__LITTLE_ENDIAN__)) || \
+                     defined(__s390x__)) || \
+                    (defined(_MSC_VER) && defined(__clang__)))
```

Comment:
could this last check be true in a hypothetical 32-bit build on windows with clang?

Reply:
which would then not have int128

Reply:
I've added a `!defined(_M_X64)` check here; would it be sufficient?